### ISSUE TITLE
Support for tslint 4.X?

### ISSUE THREAD

**Issue by kitsonk** [palantir/grunt-tslint#57]
When using the current version of `grunt-tslint` with `tslint` 4.0.0+ the following is logged to the console:

```
$ grunt
Running "tslint:src" (tslint) task
Warning: Linter is not a constructor Use --force to continue.
```

**Comment by adidahiya** [comment]
will be addressed by #56

**Comment by adidahiya** [comment]
Published v4.0.0, let us know if you have any issues.
